src/checker/http_checker.rs

```diff
@@ -173,10 +173,13 @@ fn make_trace_header(config: &CheckConfig, trace_id: &Uuid, span_id: SpanId) ->
     // according to the service's sampling policy. see
     // https://develop.sentry.dev/sdk/telemetry/traces/#header-sentry-trace
     // for more information.
+
+    let trace_id_without_dashes = trace_id.to_string().replace("-", "");
```

Comment:
I think the better way to do this would be to use the `Simple` formatter https://docs.rs/uuid/latest/uuid/fmt/struct.Simple.html

Reply:
Could just be `trace_id.simple()` and then the `Display` trait will handle things correctly.